### DISCUSSION TITLE
Expo 51 removed the `LinkProps` export from `expo-router`

### DISCUSSION THREAD

**Discussion by Rhym** [expo/router#931] [Q&A]
I don't see this in the changelogs, but `LinkProps` seems to have been removed from the router package. I'm not sure how we're supposed to get that type now for components that return a `Link` component, and require the correct types as props.

**Comment by vladimirschneider** [comment]
Hello, you can try to use ExpoRouter, it works for me

```
import { ExpoRouter } from 'expo-router/types/expo-router';

ExpoRouter.LinkProps
```

Actually, it works even  better then LinkProps 'couse of it supports my dynamic links better (i can finally ExpoRouter.Href)

I don't know works it early but it works with expo 51
